core/templates/dev/head/collection_player/collection_player.html

```diff
@@ -14,7 +14,8 @@
 
     <style>
       html, body {
-        background-color: #eee;
+        background: url('/images/general/background.png') no-repeat center center fixed;
```

Comment:
Actually this is how it's supposed to be -- see the "background-position" CSS rule docs.

Though it turns out that the other "center" is optional. Probably better to be explicit, though.